src/modules/repository/implementation/sequelize/repositories/paranet-asset-repository.js

```diff
@@ -0,0 +1,123 @@
+import Sequelize from 'sequelize';
+
+class ParanetAssetRepository {
+    constructor(models) {
+        this.sequelize = models.sequelize;
+        this.model = models.paranet_asset;
+    }
+
+    async createMissedParanetAssetRecord(missedParanetAsset, options) {
```

Comment:
We don't need to create missed reoced as all records are created when they are fetched, instead we should increment retry

Reply:
yeah, it's just stupid naming that i copied from before